### DISCUSSION TITLE
Missing Laravel official package

### DISCUSSION THREAD

**Discussion by Rattone** [HassanZahirnia/laravel-package-ocean#216] [Package Suggestions]
I noticed that [laravel/ui](https://github.com/laravel/ui) is not listed among the official packages

**Comment by HassanZahirnia** [comment]
Hey! That package is considered "Legacy" from the Laravel team and is based on Bootstrap CSS so they recommend using other products.

**Reply by Rattone** [reply]
For those who use bootstrap it is still the best starter kit. It is also still supported
